Jenkinsfile

```diff
@@ -1,7 +1,7 @@
 buildMvn {
-  publishModDescriptor = 'no'
+  publishModDescriptor = 'yes'
```

Comment:
are these changes intentional?  I didn't see them mentioned in the PR description

Reply:
PR description updated